lib/common_test/src/ct_slave.erl

```diff
@@ -309,7 +309,14 @@ is_started(ENode) ->
 
 % make a Erlang node name from name and hostname
 enodename(Host, Node) ->
-    list_to_atom(atom_to_list(Node)++"@"++atom_to_list(Host)).
+    case lists:splitwith(fun($@) -> false;
+                            (_)  -> true
+                         end, atom_to_list(Node)) of
+        {Head, [$@,_|_]=HostPart} ->
+            list_to_atom(Head ++ HostPart);
+        _ ->
+            list_to_atom(Node ++ "@" ++ Host)
```

Comment:
The previous version of the code called `atom_to_list` on `Node` and `Host`. Is that needed here as well?